docs/index.rst

```diff
@@ -1,12 +1,52 @@
 .. Tunable documentation master file.
 
+Tuneit: optimize, benchmark and crosscheck
+==========================================
+
 Welcome to Tuneit's documentation!
 ===================================
 
+
+Basic Concepts
+--------------
+The Tuneit package works with computational graphs, which have two main phases:
+
+- A construction phase, where the graph is being built. Every operation that needs to be performed will be added to the graph as node along with all the variables and data input and output. Each type of node is visualised differently in the graph as it is shown below:
+  
+  * Variables: they are represented using diamonds. The outline is red in case the variable does not have a value yet and green in case the                 variable has been assigned a fixed value.
```

Comment:
This page is fine. Just few comments here:
- use bold for Variables, Operations, Data
- provide some code to explain what you mean just above the list. E.g.
```python
# Example of Variables, Data and Operations

from tuneit import *

a = data()
x = variable(range(10))
y = variable(range(5))

axpy = a * x + y

visualize(axpy)
```